llvm/test/CodeGen/X86/GC/alloc_loop.ll

```diff
@@ -31,6 +32,8 @@ entry:
 	%B.1 = load ptr, ptr %B
 	%A.1 = load ptr, ptr %A
 	call void @llvm.gcwrite(ptr %A.1, ptr %B.upgrd.1, ptr %B.1)
+	; CHECK-NOT: call void @llvm.gcwrite(ptr %A.1, ptr %B.upgrd.1, ptr %B.1)
```

Comment:
-NOT checks are perilous and should be as lax as possible. Better would be -NEXT checks to show it's not present